spyder/utils/tests/test_programs.py

```diff
@@ -36,7 +37,39 @@ def test_run_python_script_in_terminal_with_wdir_empty(tmpdir, qtbot):
     res = outfilepath.read()
     assert res == 'done'
 
-    
+@pytest.mark.skipif(os.environ.get('TEST_CI_WIDGETS', None) == True,
+                    reason='To only run the test in spyders CI services.')
+def test_is_python_interpreter_linux_true():
+    filename = "$HOME/miniconda/bin/python"
+    valid = is_python_interpreter(filename)
+
+    assert valid == True
```

Comment:
Also the best wishes for 2017 to you, @goanpeca! and to whoever else reads this :beer: :balloon: